<!--
 Copyright 2022 Shadow Robot Company Ltd.
 This program is free software: you can redistribute it and/or modify it
 under the terms of the GNU General Public License as published by the Free
 Software Foundation version 2 of the License.
 This program is distributed in the hope that it will be useful, but WITHOUT
 ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
 FITNESS FOR A PARTICULAR PURPOSE. See the GNU General Public License for
 more details.
 You should have received a copy of the GNU General Public License along
 with this program. If not, see <http://www.gnu.org/licenses/>.
-->

<launch>
  <node name="calib_strain_gauges" pkg="strain_gauge_calibration" type="calib_strain_gauges.py" output="screen">
    <param name="initial_weight" type="int" value="0" />
    <param name="incremental_weight" type="int" value="100" />
    <param name="final_weight" type="int" value="500" />
  </node>
</launch>
